spec/requests/templates_spec.rb

```diff
@@ -28,10 +28,48 @@
   end
 
   context "editing a template" do
-    let!(:template) { FactoryBot.create(:template, :name => 'foo', :description => 'bar') }
+    let!(:template)     { FactoryBot.create(:template, :name => 'foo', :description => 'bar') }
+    let(:zone)          { FactoryBot.create(:zone, :name => "api_zone") }
+    let(:ems)           { FactoryBot.create(:ems_vmware, :zone => zone) }
```

Comment:
don't think zone is needed
```suggestion
    let(:ems)           { FactoryBot.create(:ems_vmware) }
```

if it is, then just ignore this suggestion